app/lib/language-engine.js

```diff
@@ -0,0 +1,24 @@
+import counterpart from 'counterpart';
+
+const load = (language) => {
+  const defaultLanguage = { label: 'English', value: 'en' };
```

Comment:
Good point, yes. As you said, that would accommodate people creating projects with a different default language than English.